IFC4x3/Properties/m/MaximumOperationVoltage_3k02akH0b1w9nyB3MG_zgK/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="MaximumOperationVoltage_3k02akH0b1w9nyB3MG_zgK" Name="MaximumOperationVoltage" UniqueId="ee00292e-4409-41e8-9c7c-2c3590fbda94" PrimaryDataType="IfcElectricVoltageMeasure" />
```

Comment:
Should this not be re-used? `RatedVoltage` for example.

Reply:
This can work. Resolved: https://github.com/bSI-InfraRoom/IFC-Specification/pull/165/commits/e761913e0194604f46abee1c51db7622523f3f86